modules/imgproc/test/test_resize_nn.cpp

```diff
@@ -0,0 +1,28 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+
+#include "test_precomp.hpp"
+
+namespace opencv_test { namespace {
+
+        TEST(Resize_NN, accuracy)
+        {
```

Comment:
No need to create a new file.
Please put code after this resize test: https://github.com/opencv/opencv/blob/4.2.0/modules/imgproc/test/test_imgwarp.cpp#L1403-L1414

Please add issue number into test name.

Please avoid indentation in namespaces.